day_7/high_level.md

```diff
@@ -0,0 +1,10 @@
+### Ceasar Cipher
+
+# Pseudocode steps to program:
+1. You give the cipher an input string and shift value.
+1. String is then split at every letter.
+1. Every letter in converted to a number from 1-26 based on the alphabetical order.
+1. The shift value is applied to each number, rolling back to 1 if it passes 26, or back to 26 if it goes below 1.

```

Comment:
A trick to the caesar cipher is using modulo 26 for the shift to determine the value of the new letter. I think if you can determine how to shift one letter to another, you'd get there.